docs/processing-performance.asciidoc

```diff
@@ -5,40 +5,110 @@ APM Server performance depends on a number of factors: memory and CPU available,
 network latency, transaction sizes, workload patterns,
 agent and server settings, versions, and protocol.
 
-Let's look at a simple example that makes the following assumptions:
+// Assumptions
+// TBD
 
-* The load is generated in the same region as where APM Server and {es} are deployed.
-* We're using the default settings in cloud.
-* A small number of agents are reporting.
+// Method
+To help you understand how sizing APM Server impacts performance, we tested several scenarios:
```

Comment:
Can we flip this - _help you understand how to size the APM Server so that it can keep up with the load that your Elastic APM agents  are sending._